index.html

```diff
@@ -0,0 +1,525 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" href="style.css">
+      <script src="https://kit.fontawesome.com/6e699a3632.js" crossorigin="anonymous"></script>
+      <title>Space Reveals Big Bang's Smoking gun</title>
+   </head>
+   <body>
+      <header>
+         <nav>
+            <ul class="nav_links">
+               <li><a href="#"><i class="fas fa-search"></i></a></li>
+               <li><a href="#"><i class="fas fa-bars"></i></a></li>
+               <li><a href ="#" class="Spacelink"><strong>SPACE & COSMOS</strong></a></li>
+            </ul>
+            
+         </nav>
+         <a class="logo" href="#"><img src="https://www.vectorlogo.zone/logos/nytimes/nytimes-wordmark.svg" alt="nytimeslogo"></a>  
+         <button class="btn" type="submit">SUBCRIBE NOW</button>
+         <button class= "btn" type="submit">LOG IN</button> 
+      </header>
+      <main>
+          <br><br><br><br><br>
+         <div class=flex-container>
+            <h1><small>Space Ripples Reveal Big Bang’s Smoking Gun</small></h1>
+            <figure class="fig-image">
```

Comment:
## Improve the general layout of the page
### The main Image:
- [ ] The main image covers the entire page. it should not. It must be strictly centred and its captions directly underneath it.
- [ ] Add the social links beneath the main heading text and centre the main heading (title) as shown on the original page.

|[Source page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![main-image](https://user-images.githubusercontent.com/22345961/108259483-32f41700-7172-11eb-89b3-e4eb638f3b26.png)|![main-image-clone](https://user-images.githubusercontent.com/22345961/108259506-3a1b2500-7172-11eb-9006-7dc2dde4ccf8.png)|

### The general page layout
- [ ] All the content of the page is mainly centred. What we have is left-leaning page content. Work on this.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![main-text](https://user-images.githubusercontent.com/22345961/108260097-f379fa80-7172-11eb-83ae-32c6056085f5.png)|![main-text-clone](https://user-images.githubusercontent.com/22345961/108260118-faa10880-7172-11eb-8017-f9fa32a2efee.png)|

### Editor's pick
- [ ] The `editor's pick` should be aligned right of the article. See the inspiration below.
* [ ] This section must also use the [`<aside>`](https://www.w3schools.com/tags/tag_aside.asp) tag for `Editor's picks` section on the righmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![editors-pick](https://user-images.githubusercontent.com/22345961/108260498-73a06000-7173-11eb-8d10-abe828c93077.png)|![editors-pick-clone](https://user-images.githubusercontent.com/22345961/108260525-7ac76e00-7173-11eb-952a-480cac3e7547.png)|

### Fix Theory of inflation section
- [ ] A good way to have this sorted out would be to use CSS `GRID` and `FLEX` rules to accomplish this. See below for inspiration.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![theory-of-inflation](https://user-images.githubusercontent.com/22345961/108261204-5a4be380-7174-11eb-85ae-0bf5034638cc.png)|![theory-of-inflation-clone](https://user-images.githubusercontent.com/22345961/108261220-5fa92e00-7174-11eb-810d-9c84a02b6bef.png)|


### Footer 
- [ ] The footer will also require some attention. While at it, kindly ensure the page has a fixed part of the `footer` at the bottom.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JulianaOsemeke/NewYorkTimesArticle/Solo-Project/index.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/108261476-bf073e00-7174-11eb-9050-06866201bec1.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/108261489-c4fd1f00-7174-11eb-840c-30cf087b958e.png)|